src/components/SingleArticle.jsx

```diff
@@ -24,6 +25,17 @@ const SingleArticle = () => {
     });
   }, [article_id]);
 
+  const [voteChange, setVoteChange] = useState(0);
+  function handleVote(vote) {
+    //  event.preventDefault();
+    setVoteChange((currChange) => {
+      return currChange + vote;
+    });
+    updateVotes(article_id, vote).then((updatedArticle) => {
+      setArticle(updatedArticle);
```

Comment:
Ideally we want some logic to handle if the patch fails. This should ideally not increment/decrement the vote but also provide feedback to the user, like a message 'not possible to vote at this time'